### DISCUSSION TITLE
Bubble Label - name change

### DISCUSSION THREAD

**Discussion by finnok** [ulic75/power-flow-card#162] [Q&A]
Hi, newbe here and luvin your work.   How do I change the bubble name from Gas to EV ??  - otherwise all working nicely

Thank you

**Comment by ulic75** [comment]
This feature is not available in this card.  You would have to edit the js file by hand.

**Comment by finnok** [comment]
Perfect , I wont chance that ... Thank you for responding 

Brian.
